pom.xml

```diff
@@ -132,6 +132,11 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
```

Comment:
Do we need to have this dependency now as we are not using any libraries?

Reply:
Yes this is required to extract commit id details